es/separable_natural_es.py

```diff
@@ -1,13 +1,39 @@
+import glob
 import multiprocessing as mp
 import numpy as np
+import pickle
 from . import lib
 
+
+def load_checkpoint():
```

Comment:
What about enabling the user to specify a specific checkpoint, not only the most recent one? You could implement this by allowing `load_existing_checkpoint` to be specified as `True` or `-1` (for the most recent one) or as an integer specifying the generation.